pyproject.toml

```diff
@@ -182,9 +178,7 @@ dev = [
   "trio",
   "aiosqlite",
   "asyncpg>=0.29.0",
-  "psycopg[pool,binary]>=3.1.10,<3.2; python_version < \"3.13\"",
-  "psycopg[pool,c]; python_version >= \"3.13\" and sys_platform == 'linux'",
-  "psycopg[pool]; python_version >= \"3.13\" and sys_platform != 'linux'",
```

Comment:
Are these constraints not needed anymore?

Reply:
Nope.  They have built the necessary binaries for 3.13  and Windows now.